### PR TITLE
vocascan-frontend "Themes" section

### PR DESCRIPTION
**Pull request by Tracer1337**
Documentation for the new themes feature: https://github.com/vocascan/vocascan-frontend/pull/111

### CODE REVIEW

File: docs/vocascan-frontend/themes.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+:root {
+    --background-color: #fff;
+    /* Further configuration... */
+}
+```
+
+Mounting the ``themes/`` directory:
+
+```bash
+docker run -v ./themes:/usr/share/nginx/html/themes ...
```
Comment (luwol03) [inline docs/vocascan-frontend/themes.md]: afaik this command wont work. You cant use relative paths via docker run. You need something like `$(pwd)/themes` but I suggest changing that to an docker-compose diff what you need to change.

File: docs/vocascan-frontend/themes.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+### Example
+
+Folder structure:
+
+```
+|-- themes
+|   |-- my-theme.css
+```
+
+Example theme (``my-theme.css``):
```
Comment (luwol03) [inline docs/vocascan-frontend/themes.md]: why have you used doubbl ` ?

**Review by luwol03 — CHANGES_REQUESTED** [review]

File: docs/vocascan-frontend/themes.md (hunk 1 above)
Comment (Tracer1337) [reply docs/vocascan-frontend/themes.md]: This does work, I used it for testing

File: docs/vocascan-frontend/themes.md (hunk 2 above)
Comment (Tracer1337) [reply docs/vocascan-frontend/themes.md]: Fixed

File: docs/vocascan-frontend/themes.md (hunk 1 above)
Comment (luwol03) [reply docs/vocascan-frontend/themes.md]: ok. I havent said anything :D - but can we still change that to docker-compose?

Comment (Tracer1337) [reply docs/vocascan-frontend/themes.md]: Done

**Comment by luwol03** [comment]
Can you please rebase your changes on top of the experimental branch.

**Comment by Tracer1337** [comment]
> Can you please rebase your changes on top of the experimental branch.

Done

**Review by trite8q1 — APPROVED** [review]

**Comment by Tracer1337** [comment]
@luwol03 I think we can merge this

**Comment by luwol03** [comment]
We'll merge this after the Frontend pr.

**Comment by luwol03** [comment]
Can you also add the config file mounting part to the docs for me?

**Comment by Tracer1337** [comment]
> Can you also add the config file mounting part to the docs for me?

Yes, but make a new ticket for this

File: docs/vocascan-frontend/configuration.md [hunk 3]
```diff
@@ -3,8 +3,35 @@
 You can configure vocascan-frontend with the following options. For a instruction how to use them look at the individual
 [installation guides](vocascan-frontend/installation/installation.md).
 
+## Environment Variables
```
Comment (luwol03) [inline docs/vocascan-frontend/configuration.md:6]: can we keep this more general and then describe afterwards, that `REACT_APP_VOCASCAN_` + option is needed for build, `VOCASCAN_` + option is needed for docker start and then how config files work? Then there is no need to also mention this again and again.

File: docs/vocascan-frontend/configuration.md [hunk 4]
```diff
@@ -3,8 +3,35 @@
...
+
+The following options can be written into the existing `public/config.js` file:
+
+```
+ENV: string
+BASE_URL: string
+SHOW_PLANS: boolean
+THEME_SELECT: string
+themes: Object
+```
```
Comment (luwol03) [inline docs/vocascan-frontend/configuration.md:25]: This can then be removed

**Pull request closed without merging**